CHANGELOG/unreleased/kong/11484.yaml

```
@@ -0,0 +1,7 @@
+message: "Tracing: fix an issue that resulted in some parent spans to end before their children due to different precision of their timestamps"
+type: bugfix
+scope: PDK
+prs:
+  - 11484
+jiras:
+  - "KAG-2336"
```

It's ok to go without this change.

```suggestion
  - "KAG-2336"
issues:
  - 11294
```